pyanaconda/modules/timezone/installation.py

```diff
@@ -16,6 +16,7 @@
 # Red Hat, Inc.
 #
 import os.path
+
```

Comment:
Please change the commit message of a1f72e83265a75e8da6a7ff6d69a557f22c20f8c.
You are writing there a reason but commit message should contain what it does not why is it doing.

Reply:
Now in 1fb43b54e5c7f7fdbe0cab6e3ad86bdb7324e8b7 it says  Fix code style - conventions, spacing, import order